src/utils.py

```diff
@@ -80,6 +77,28 @@ def close_instagram(device_id):
     os.popen("adb" + ("" if device_id is None else " -s " + device_id) +
              " shell am force-stop com.instagram.android").close()
 
+def random_sleep():
+    delay = randint(2, 6)
+    print("Sleep for " + str(delay) + (delay == 1 and " second" or " seconds"))
+    sleep(delay)
+
+def screen_care():
+    System_OS = platform.system()
+    print(f"You're on {System_OS}!")
+    if "Linux" == System_OS:
+        status=os.popen ("adb shell dumpsys input_method | grep mInteractive=")
+    else:
+        status=os.popen ("adb shell dumpsys input_method | findstr mInteractive=")
```

Comment:
I think it's better to make it environment independent. You can extract the whole output from 'adb shell dumpsys input_method' and grep the result in python.

Reply:
ok fine!